snapcraft/tests/bin/readelf/ldd

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```

Comment:
Is there a reason this is nested under a 'readelf' directory instead of just being in bin?

Reply:
I should call this topdir elf